src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1132,6 +1134,17 @@ protected void processCopyLines(List<String> dockerfileLines, String buildContex
             }
         }
     }
+    
+    /**
+     * 
+     * The intention here is to give a chance to run logic for an app update
+     * but not necessarily a full re-deploy
+     * 
+     * @throws PluginExecutionException
+     */
+    protected void updateLooseApp() throws PluginExecutionException {
+    	// no-op - placeholder to override
```

Comment:
If this is meant to be overridden in ci.maven, can you change this to an `abstract` method?  (This will mean it needs to be overridden in ci.gradle as well, but that can be a no-op if it's not applicable for Gradle).

Reply:
Done